bylaws.md

```diff
@@ -1,14 +1,17 @@
 <!-- multilingual suffix: en, da, se -->
-<!-- [common] -->
-This document has multiple purposes. First, it functions as  the bylaws for a local chapter as required by the ACM. Secondly, it functions as official documentation required to register the organisation in both the danish and swedish business register.
-
-The document is also an exercise in pratical multilingualism and cross-border collaboration.
+<!-- [en] -->
+# Bylaws
+This document contains the bylaws for a local chapter of Association for Computing Machinery, hereafter referred to as ACM. In addition this document contains the information required for the registration of voluntary organisations in the danish and swedish business registers.
 
 <!-- [en] -->
 # Name and location
-The name of the organisation is Oeresund Chapter of Association for Computing Machinery, sometimes reffered to as ACM, and is also using the local names ACM Øresund og ACM Öresund.
+The name of the organisation is Oeresund Association for Computing Machinery Chapter and is using the local names of ACM Øresund and ACM Öresund. Hereafter referred to as The Chapter.
```

Comment:
The name format is specified by ACM https://www.acm.org/chapters/chapter-policies